src/webpack.config.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    entry: [
+        './react/index.jsx'
```

Comment:
Webpack is what creates the `bundle.js` that loads all of our code onto the client's machine. Not sure what this `entry` is, but if it's a list of everything that needs to be bundled then maybe we want to just bundle the whole folder. Also not sure what to name this folder. `/react/` for now